ext/hash/tests/hash_update_stream_basic_001.phpt

```diff
@@ -0,0 +1,37 @@
+--TEST--
+Ensure hash_update_stream() always returns the same hash when $length = 0
+--CREDITS--
+Symeon Charalabides <symeon@systasis.com> - @phpdublin
+--SKIPIF--
+<?php
+if (phpversion() < "5.3.0") {
+    die('SKIP - php version too low.');
+}
+require_once(dirname(__FILE__) . '/skip_mhash.inc'); ?>
+?>
+--FILE--
+<?php
+
+for ($j=0; $j<3; $j++)
+{
+  // Create pseudo-random hash
+  $bytes = openssl_random_pseudo_bytes(15, $cstrong);
```

Comment:
You should either use `random_bytes` or add a skip if openssl is not loaded...

Reply:
Added skip for openssl and another for hash.